Utils/github_workflow_scripts/utils.py

```diff
@@ -203,3 +205,33 @@ def get_content_reviewers(content_roles: dict[str, Any]) -> tuple[list[str], str
     except KeyError as ke:
         print(f"Error parsing reviewers: {str(ke)}.")
         sys.exit(1)
+
+
+def get_doc_reviewer(content_roles: dict[str, Any]) -> str | None:
+    """
+    Retrieve the doc reviewer from content roles JSON/`dict`.
+
+    Args:
+        - `content_roles` (``dict[str, Any]``): The current content team roles and members.
+
+    Return:
+        - `str` of document reviewer GitHub username.
+        If there's an error in retrieving the tech writer/doc reviewer,
+        an `None` is returned.
+    """
+
+    try:
+        doc_reviewer: str = content_roles[DOC_REVIEWER_KEY]
+
+        if not isinstance(doc_reviewer, str) or not doc_reviewer:
+            print(f"'{DOC_REVIEWER_KEY}' is not a string. Terminating...")
+            return None
+
+        if not DOC_REVIEWER_KEY or not DOC_REVIEWER_KEY:
+            print(f"No '{DOC_REVIEWER_KEY}' key specified or is empty")
+            return None
+
+        return doc_reviewer
+    except KeyError as ke:
+        print(f"Error parsing doc reviewer: {str(ke)}.")
+        return None
```

Comment:
Replace this whole method with
```python
if not (reviewer := content_roles.get(DOC_REVIEWER_KEY)):
    raise ValueError("Cannot get doc reviewer")
return reviewer
```

Reply:
TIL of how to use the Walrus Operator :)